screengrab/screengrab-lib/src/main/java/tools.fastlane.screengrab/locale/LocaleUtil.java

```diff
@@ -25,8 +25,12 @@ public static void changeDeviceLocaleTo(Locale locale) {
             methodGetDefault.setAccessible(true);
             Object activityManagerNative = methodGetDefault.invoke(amnClass);
 
-            Class iam = Class.forName(activityManagerNative.getClass().toString());
-            Method methodGetConfiguration = iam.getMethod("getConfiguration");
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
```

Comment:
@vanniktech I think we've fixed this now, would you might giving it one more try 🙏?